### ISSUE TITLE
Hide tray icon at runtime

### ISSUE THREAD

**Issue by KyleTheScientist** [spyoungtech/ahk#270]
First off, thanks for developing this library. AHK is so powerful but the scripts are so goddamn ugly. You've done a great job bringing that power over to Python.

As you know, the tray icon that spawns for the main process likes to hang around after the process ends until you hover over it. I'd like to be able to hide the tray icon at runtime to solve the issue. I'm aware of the `NoTrayIcon` directive, but I do actually want to use the icon to communicate information about the state of the process. 

Here's a sort of stripped down demo of the use-case:
```python
import time
from threading import Event
from ahk import AHK
from ahk.directives import NoTrayIcon


ahk = AHK(directives=[NoTrayIcon(apply_to_hotkeys_process=True)]) # Stop child processes from showing a tray icon
ahk.menu_tray_tooltip('Temporary Tray Icon')
ahk.menu_tray_icon('default.ico')
ahk.menu_tray_icon_show()

# Use the tray icon to reflect which macro layer we're on
ahk.add_hotkey('~Control', lambda: ahk.menu_tray_icon('control.ico')) 
ahk.add_hotkey('~Shift', lambda: ahk.menu_tray_icon('shift.ico')) 
ahk.add_hotkey('~^Alt', lambda: ahk.menu_tray_icon('alt.ico')) 

ahk.add_hotkey('~Control up', lambda: ahk.menu_tray_icon('default.ico')) 
ahk.add_hotkey('~Shift up', lambda: ahk.menu_tray_icon('default.ico')) 
ahk.add_hotkey('~Alt up', lambda: ahk.menu_tray_icon('default.ico')) 

# Stop hotkeys with Ctrl+Q
soft_stop = Event() # To negotiate a hault from the hotkey thread
ahk.add_hotkey('^q', lambda: soft_stop.set())
ahk.start_hotkeys()

while not soft_stop.is_set():
    time.sleep(1)

# Hide the tray icon here so it won't stick around after the script exits.
ahk.stop_hotkeys()
ahk.menu_tray_icon_hide()
print('Exiting.')
```

I couldn't really find any documentation for how to do this even in a .ahk script, so I'm not sure it's even possible to do without some system call black magic. Still, it would be nice.

**Comment by spyoungtech** [comment]
Thanks for opening this. I believe this is supported in AutoHotkey, according to [this documentation](https://www.autohotkey.com/docs/v1/lib/Menu.htm#NoIcon). 

I'm not sure if this is directly exposed in this library yet. But since there is a function for showing the icon, it stands to reason we should have a function for hiding it, too.

**Comment by spyoungtech** [comment]
@KyleTheScientist this is now released and available in version 1.5.2

Let me know if that works for you.

**Comment by KyleTheScientist** [comment]
You're a legend. Works great, thanks!
